Source/Game.cs

```diff
@@ -0,0 +1,50 @@
+﻿namespace GameOfLife.Source
+{
+    /// <summary>
+    /// Class for creating new game instance.
+    /// </summary>
+    public class Game
+    {
+        private Field field;
+        private Cells cells;
+        public bool shouldRun = false;
+
+        /// <summary>
+        /// Function that processes game and its logic.
+        /// </summary>
+        public void Play()
+        {
+            const string fieldWidthInfo = "Enter game field width (4 - 120 characters long)";
+            const string fieldHeightInfo = "Enter game field height (4 - 35 characters long)";
+            ushort fieldWidth = UserInterface.ProcessInput(fieldWidthInfo, 4, 120);
```

Comment:
default property values must be moved separate file so it is not "magic number"

Reply:
fixed